test/controllers/giving_profiles_controller_test.rb

```diff
@@ -0,0 +1,9 @@
+require 'test_helper'
+
+class GivingProfilesControllerTest < ActionDispatch::IntegrationTest
+  test "should get index" do
+    get giving_profiles_index_url
```

Comment:
think this needs to be updated to reflect new route. May just be easier to regen the test